README.md

```diff
@@ -82,4 +82,20 @@ Wait for a few seconds for Apache Unomi to start, you can then open up your brow
 ## Testing procedure
 
 Once the environment is started, the tracker can be verified following this procedure:
- - To be updated
+ 
+ **Test1 : page visit increment**
+ In Unomi API check the number of views for index.html
```

Comment:
In the example, could you add the curl calls that can be used for these checks ? (this one and below)